.github/workflows/power_eflow_vm.yml

```diff
@@ -0,0 +1,35 @@
+name: Turn EFLOW vm ON or OFF
+
+on:
+  workflow_dispatch:
+    inputs:
+      power:
+        description: 'Turn the VM ON or OFF'
+        required: true
+        type: boolean
+      vm_name:
+        description: 'The name of the vm to be turned on'
+        required: true 
+        default: 'eflow'
+        type: string
+      rg_name:
+        description: 'The name of the resource-group the VM belongs to'
+        required: false
+        type: string
+        default: 'lorae2etests'
+
+jobs:
+  power_azure_vm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Power Azure VM
+        uses: ./.github/actions/power-azure-vm
+        with:
+          AZURE_VM_NAME: ${{ github.event.inputs.vm_name }}
+          AZURE_RG_NAME: ${{ github.event.inputs.rg_name }}
+          POWER_SWITCH: ${{ github.event.inputs.power }}
+          AZURE_SP_CLIENTID: ${{ secrets.AZURE_SP_CLIENTID }}
+          AZURE_SP_SECRET: ${{ secrets.AZURE_SP_SECRET }}
+          AZURE_TENANTID: ${{ secrets.AZURE_TENANTID }}
```

Comment:
I think you need to add a carriage return at the end of each file, otherwise Atif will get angry :D